stable/application-mesh/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.14
+version: 0.3.15
```

Comment:
It is good practice to update minor/major versions when you make big changes. 
- minor - when interface hasnt changed by something significant is happening under the hood
- major - when required attributes interface changed or when your changes are big enough to call them major